### DISCUSSION TITLE
Licensing question (adding additional seats)

### DISCUSSION THREAD

**Discussion by yaroz** [rustdesk/rustdesk-server-pro#330] [Q&A]
If I buy a license for 400 managed clients, and we go over that, can we add another 100, or are we cut off until the next license cycle?

**Comment by rustdesk** [comment]
Yes, https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/license/#renewupgrade-license

**Reply by yaroz** [reply]
@rustdesk Great.  Thank you.  Can we purchase for multiple years, to lock in the rate?  I don't see that option on the pricing page.

**Reply by rustdesk** [reply]
Actually you can, you can "renew" on https://rustdesk.com/self-host/account/#/license, every time, you renew, additional one year is added.
